### DISCUSSION TITLE
Is there a standard place to store data?

### DISCUSSION THREAD

**Discussion by wsw70** [nuxt/nuxt#29941] [Questions]
Brand new to Nuxt, I tried to find in the docs (which are very good BTW) where I am expected to store data (a SQLite db, some JSON files I need to access).

Is there a standard place to store them in a Nuxt project directory structure?

**Comment by TheAlexLichter** [comment]
If you want to store data that isn't relational, you can use the [server assets from nitro](https://nitro.unjs.io/guide/assets#server-assets) + your preferred KV storage driver for it.

If not, you can use db0 to connect to your favorite DB, an ORM like drizzle or prisma, or a more managed solution like supabase
